test/main.js

```diff
@@ -241,6 +241,30 @@ describe('rpi-gpio', function() {
                 });
             });
 
+            context('and direction is specified low-ward', function() {
```

Comment:
tests that DIR_HIGH and DIR_LOW do indeed set the pin's direction to 'high' and 'low' respectively.